tests/integration/test_fim/test_files/test_follow_symbolic_link/data/test_cases/cases_disabled.yaml

```diff
@@ -2,88 +2,118 @@
   description: When a real-time monitored directory es deleted
                or added FIM inmediately raises a log.
   configuration_parameters:
-    TEST_DIR: !!python/object/apply:os.path.join [/test_dir, symlink]
+    TEST_DIR: !!python/object/apply:os.path.join
+          args: [!!python/object/apply:os.getcwd [],test_dir, symlink]
```

Comment:
add a blank space after the first comma "[], test_dir".

Reply:
Done!